src/RwaLiquidationOracle.sol

```diff
@@ -132,7 +132,8 @@ contract RwaLiquidationOracle {
     // --- liquidation check ---
     // to be called by off-chain parties (e.g. a trustee) to check the standing of the loan
     function good(bytes32 ilk) external view returns (bool) {
-        require(ilks[ilk].pip != address(0));
-        return (ilks[ilk].toc == 0 || add(ilks[ilk].toc, ilks[ilk].tau) < block.timestamp);
+        require(ilks[ilk].pip != address(0), "RwaOracle/unknown-ilk");
+        // tell not called or still in remediation period
+        return (ilks[ilk].toc == 0 || block.timestamp < add(ilks[ilk].toc, ilks[ilk].tau));
```

Comment:
@livnev and this too.  This is what we want?

Reply:
The new version is correct, yes: the position is `good()` if there is not an outstanding `tell` or if the remediation period has not elapsed yet.